packages/fxa-shared/test/metrics/amplitude.js

```diff
@@ -138,6 +140,7 @@ describe('metrics/amplitude:', () => {
 
       it('returned the correct result', () => {
         assert.deepEqual(result, {
+          app_version: '0',
```

Comment:
From the discussion at https://github.com/mozilla/fxa/pull/863#discussion_r277979928:

> Side effect of the monorepo. Previously, the `../../../package.json` in `metrics/amplitude.js` would only resolve to something when run from `node_modules` in one of our consuming services, so `app_version` would not be set when the tests were run locally and in CI.
> 
> But now it resolves to the top-level `package.json`, which has `0` in the train number position.

Reply:
Dammit, it fails when the tests are run in a container of course. Changing the assertion.

Reply:
Fixed.